src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -1444,7 +1444,7 @@ def test_aks_nodepool_get_upgrades(self, resource_group, resource_group_location
                  checks=[
                      # if rerun the recording, please update latestNodeImageVersion to the latest value
                      self.check_pattern('latestNodeImageVersion',
-                                'AKSUbuntu-1804gen2containerd-2021.(0[1-9]|1[012]).(0[1-9]|[12]\d|3[01])'),
+                                'AKSUbuntu-1804gen2containerd-202([0-9]).(0[1-9]|1[012]).(0[1-9]|[12]\d|3[01])'),
```

Comment:
If Version == 'AKSUbuntu-1804gen2containerd-2020.01.01'，it will match the pattern and check as latestNodeImageVersion，is that what you expect?

Reply:
Yes, `AKSUbuntu-1804gen2containerd-2020.01.01` should be a valid version string.